copy-secrets.sh

```diff
@@ -0,0 +1,3 @@
+touch secrets.js
+echo "window.RUDDERSTACK_HT_WRITE_KEY = \"$RUDDERSTACK_HT_WRITE_KEY\";" > secrets.js
```

Comment:
1 case, secret not found what will happen then.
eg -> in perf/devstack we wont be adding this rigth, so u want to add dummy values in secret or handle it in code 
(best would be to add dummy, so code remains same)

Reply:
We need a way to first check if analytics(rudderstack) is enabled and then execute this. Else, for those who don't want this, it might end up failing.

Reply:
Will add the feature flag kinda stuff so it can be dynamically enabled and disabled.

Reply:
Added feature flag for analytics